etc/scripts/generate-github-test-data.py

```diff
@@ -110,18 +155,24 @@ def create_workflow_file():
         with open(workflow_file_path, 'w') as workflow_file:
             workflow_file.write(content)
 
-    os.system(f'git add .github/workflows/{workflow_file_name}')
+    subprocess.run(f'git add .github/workflows/{workflow_file_name}', shell=True, check=True)
 
 def create_pull_request(repo_name, branch_name):
-    create_pull_request_url = f'{BASE_URL}/repos/{USERNAME}/{repo_name}/pulls'
+    create_pull_request_url = f'{BASE_URL}/repos/{ORGANIZATION_NAME}/{repo_name}/pulls'
+    pull_request_data = {
+        'title': f'{args.issue_prefix}-{random.randint(100, 999)}: Pull request',
```

Comment:
I know these issue keys dont match, it no matter... but looks weird when reading